Library/Homebrew/cmd/test-bot.rb

```diff
@@ -639,10 +639,9 @@ def homebrew
         tests_args << "--coverage" if ruby_two && ENV["TRAVIS"]
         test "brew", "tests", *tests_args
         test "brew", "tests", "--no-compat"
+        test "brew", "readall", "--syntax" if ruby_two
       elsif @tap.core_tap?
-        readall_args = []
-        readall_args << "--syntax" if ruby_two
-        test "brew", "readall", "--aliases", *readall_args
+        test "brew", "readall", "--aliases"
```

Comment:
I realize this is not directly related to the topic of this PR, but I wonder if we even need to distinguish between `@tap.core_tap?` and all other `!@tap.nil?` cases. (Not passing the tap name currently means to check all currently tapped taps, something that could be wasteful, though not a lot.)